client/src/components/Image.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import { withStyles } from '@material-ui/core'
+import DeleteOutlinedIcon from '@material-ui/icons/DeleteOutlined';
+
+const styles = {
+  deleteIconContainer: {
+    position: 'absolute',
+    height: '75px',
+    width: '75px',
+    backgroundColor: 'grey',
+    opacity: '50%',
+    borderRadius: '10px',
+    display: 'flex',
+  },
+  deleteIcon: {
+    height: '30px',
+    width: '30px',
+    margin: 'auto',
+    opacity: '100%'
+  },
+  imageContainer: {
+    margin: '4px',
+    maxHeight: '75px',
+  },
+  image: {
+    maxHeight: '75px',
+    maxWidth: '75px',
+    borderRadius: '10px',
+    cursor: 'pointer'
+  }
+}
+
+const Image = (props) => {
+  const { classes, image, deleteImage } = props;
+  const [isHovering, setHovering] = useState(false);
+
+  const handleMouseHover = () => {
+    setHovering(toggleHoverState)
+  }
+
+  const toggleHoverState = () => {
+    setHovering(!isHovering)
+  }
+
+  // const opacityStyle = () => {
+  //   return isHovering ? { opacity: '75%'} : { opacity: '100%' };
+  // }
```

Comment:
You can set hover styles in the styles/css by doing something like the following, and :hover is a css selector so in css you could do `h1:hover {...}`, and in css-in-js something like: `'&:hover': {opacity: '75%'}`

Reply:
I'm going to keep the logic of my hovering since I use it to display a DeletedOutlinedIcon from material-ui

Reply:
DeletedOutlinedIcon?

Reply:
Yeah!